libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.java

```diff
@@ -475,6 +475,128 @@ public Builder addWaypointNames(@Nullable String... waypointNames) {
       return this;
     }
 
+    /**
+     * The type of bicycle, either `Road`, `Hybrid`, `City`, `Cross`, `Mountain`. The default type
+     * is Hybrid.
+     * <p>
+     * This option shouldn't be used with the server-side API. It's only available for offline and
+     * it'll make the request fail if added when in online. Although, it can definitely be added to
+     * construct {@link NavigationRoute} and used to get the URL for offline.
+     *
+     * @param bicycleType the type of bicycle
+     * @return this builder for chaining options together
+     * @since 0.22.0
+     */
+    public Builder addBicycleType(@DirectionsCriteria.BicycleType @Nullable String bicycleType) {
```

Comment:
> This option shouldn't be used with the server-side API.

Should we create something like `OfflineNavigationRoute` that includes these options in the `Builder` to be more clear here than just the javadoc?